convert_wav_to_mp3.py

```
@@ -0,0 +1,76 @@
+            "ffmpeg",
+            "-y", # Overwrite output files without asking
+            "-i", input_path,
+            "-vn",
+            "-ar", "44100",
+            "-ac", "2",
+            "-b:a", bitrate,
+            output_path
+        ]
+        
+        try:
+            subprocess.run(command, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)